experiments/sphere_held_suarez_rhoe_equilmoist/observation_map.jl

```diff
@@ -27,17 +27,17 @@ function latitudinal_avg(arr)
     return dropdims(mean(arr; dims); dims)
 end
 
-function height_avg(arr)
+function height_avg(arr)  # unused
```

Comment:
Do we want to remove these unused functions entirely? In the future we should be able to use functions defined within ClimaAnalysis (which should also remove the need for hardcoded `dims` info here)

Reply:
These can be removed. I will switch this example over to ClimaAnalysis in a later PR